evap/static/scss/_adjustments.scss

```diff
@@ -133,3 +133,25 @@ $alert-colors: (
     border-color: $input-focus-border-color;
     box-shadow: $input-focus-box-shadow;
 }
+
+// Change .btn-group outlined border style to fix double borders with outlined button style
+// see https://github.com/twbs/bootstrap/issues/25888
+.btn-group.outline-fix {
+
+    // If .btn is not the last element in the group, remove the right border.
+    // Also uses :has to not remove the right border if the next element is a disabled input
+    .btn:not(:last-child):not(.dropdown-toggle):has(+ :not(.btn-check[disabled])), .btn-group:not(:last-child) > .btn {
```

Comment:
As far as I can tell, `.outline-fix .btn.dropdown-toggle` never matches on evap. I also don't know what it is supposed to fix